res/res_respoke/respoke_endpoint.c

```diff
@@ -43,6 +43,60 @@
 #include "include/respoke_app.h"
 #include "include/respoke_sdk_header.h"
 
+#define DEFAULT_STATE_BUCKETS 53
+
+static AO2_GLOBAL_OBJ_STATIC(endpoint_states);
```

Comment:
Is there any particular reason this is a global ao2 obj instead of just a normal container?